exporter/coralogixexporter/metrics_client.go

```diff
@@ -79,7 +79,7 @@ func (e *exporter) start(_ context.Context, host component.Host) (err error) {
 	headers["Authorization"] = "Bearer " + e.config.PrivateKey
 	e.metadata = metadata.New(headers)
 	e.callOptions = []grpc.CallOption{
-		grpc.WaitForReady(e.config.GRPCClientSettings.WaitForReady),
+		grpc.WaitForReady(e.config.Metrics.WaitForReady),
```

Comment:
Does this change need to be made in the trace client as well?

Reply:
Nope, this was a minor bug only in metrics client, we have 3 different configs:
```
e.Config - grpc trace settings
e.Metrics - grpc metrics settings
e.Logs - grpc logs settings
```

so the tracing one is actually using correct option :)

Reply:
ah I see... `e.Config` = trace settings was confusing me :)